packages/components/modules/notifications/NotificationsList/NotificationItem/__storybook__/mockResolvers.ts

```diff
@@ -0,0 +1,28 @@
+export const mockResolvers = {
+  Node: () => ({
+    id: 'Tm90aWZpY2F0aW9uOjIyNQ==',
+    unread: true,
+    pk: 225,
+    timestamp: '2024-10-25T19:46:09.288290+00:00',
+    level: 'INFO',
+    verb: 'COMMENTS.COMMENT_REPLY_CREATED',
+    description: 'Someone replied to your comment 1.',
+    data: {
+      extra: {},
+    },
+    actor: {
+      __typename: 'Profile',
+      id: 'UHJvZmlsZToxNTM=',
+    },
+    target: {
+      id: 'Q29tbWVudDoyMzc=',
+      __typename: 'Comment',
+    },
+    actionObject: {
+      id: 'Q29tbWVudDoyNDg=',
+      __typename: 'Comment',
+      body: 'This is the first comment reply.',
```

Comment:
![image](https://github.com/user-attachments/assets/857b98be-057d-4e79-b543-0d8f38748a3c)
look like body is not appearing on the story

Reply:
It seems we have some bug here, @anicioalexandre tried to help me, but we decided to left like this and work on a bug later